### DISCUSSION TITLE
Block access to the webpanel over the ip

### DISCUSSION THREAD

**Discussion by mrcsdl** [tobychui/zoraxy#445] [General]
hey i wonder if it is possible to block the access to the panel over ip+port.

i want that the access to the webpanel is only possible over the dns name and if possible authelia

**Comment by mrcsdl** [comment]
p.s. iam hosting zoraxy on a cloud machine with an public ip adress. and i want to block the access to the webinterface over ip+port and only allow it via zoraxy.domain.com

**Reply by tobychui** [reply]
Use startup parameter `-port=127.0.0.1:8000` and setup a routing rule to proxy `zoraxy.domain.com -> 127.0.0.1:8000`

**Comment by mrcsdl** [comment]
Thanks alot
